business_requirement_deliverable/models/business.py

```diff
@@ -138,8 +138,9 @@ class BusinessRequirementDeliverable(models.Model):
         ondelete='cascade',
         required=True
     )
-    unit_price = fields.Float(
-        string='Sales Price'
+    sale_price_unit = fields.Float(
```

Comment:
I would like to avoid unnecessary migration and would rather stick to original naming. Not ideal but good enough